.dockerignore

```diff
@@ -1,17 +1,18 @@
-.vs
-.vscode
-bin
-[_][Bb]uild
-Builds/*
-Testing/*
-.idea
-cmake-build-debug
-cmake-build-release
-build
-external/*
-!external/CMakeLists.txt
+# Ignore everything by default
+*
 
-README.md
-CHANGELOG.md
-CONTRIBUTING.md
-LICENSE
+# First-order allow exception for select directories
+!/.clang-format
+!/.githooks
+!/CMakeLists.txt
+!/Dockerfile
```

Comment:
@msz-rai , I forgot to remove this line. The Dockerfile can of course be ignored, so that changes it itself don't break the build cache after the `COPY . . ` directive.